game/game.css

```diff
@@ -2,11 +2,11 @@ body{
     background-color:gray;
 }
 
-.mapcontainer{
+.map--container{
```

Comment:
nazwy klas powinny byc raczej odseparowanie jednym '-'
w BEM korzystamy z '--' jako modifikatorów a raczej container nie jest modifikatorem

Reply:
corrected to map__container